modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -463,7 +462,7 @@ class Form extends Component {
 					/>
 				</LayoutProviderTag>
 
-				<div class="container-fluid-1280">
+				<ClayLayout.ContainerFluid>
```

Comment:
hey @eduardoallegrini, I think we can't use the React component in Metal+JSX here 😞